1_clusters/binary-auth.tf

```diff
@@ -0,0 +1,118 @@
+resource "random_pet" "keyring-name" {
+}
+
+resource "null_resource" "resource-to-wait-on" {
+  provisioner "local-exec" {
+    command = "sleep 60" # one minute to allow eventual consistency of APIs avoiding failures downstream
+  }
+  depends_on = [module.project-services]
+}
+
+resource "google_kms_key_ring" "keyring" {
+  name     = "attestor-key-ring-${random_pet.keyring-name.id}"
+  location = var.keyring-region
+  lifecycle {
+    prevent_destroy = false
```

Comment:
This can be excluded if it's false.

Reply:
Will be updated shortly, didn't see in the docs what the default was, so left it explicit more for documentation purposes to acknowledge that the kms keyring will be removed